modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts

```diff
@@ -166,9 +166,10 @@ const StyleSheetLoader = (documentOrShadowRoot: Document | ShadowRoot, settings:
     // Start loading
     const styleElem = SugarElement.fromTag('style', doc.dom);
     Attribute.setAll(styleElem, {
-      rel: 'stylesheet',
-      type: 'text/css',
-      id: state.id
+      'rel': 'stylesheet',
+      'type': 'text/css',
+      'id': state.id,
```

Comment:
These don't need to be string keys, only the key with `'` in it 🤔 
```suggestion
      rel: 'stylesheet',
      type: 'text/css',
      id: state.id,
```

Reply:
eslint complained at me as it either wants the keys to be all wrapped in `'` or none of them